docs/en/serverless/transclusion/host-details.mdx

```diff
@@ -99,5 +101,36 @@ You can also select **Actions** → **Show in Inventory** to view the host Inven
 
 </DocAccordion>
 
-{/* TODO: Find out if OSQuery tab will be included in serverless. It does not currently appear in serverless builds */}
+<DocAccordion buttonContent="Osquery">
+
+{/* Roles role="Admin" goal="use this feature"  */}
```

Comment:
@roshan-elastic For serverless, do you know what role is required to use Osquery? Is it Admin? Editor?

Reply:
@chrisdistasio @smith - You don't happen to know do you?